suisa_sendemeldung/suisa_sendemeldung.py

```diff
@@ -285,6 +286,31 @@ def get_artist(music):
     return artist
 
 
+def get_isrc(music):
+    """Get a valid ISRC from the music record or return an empty string."""
+    isrc = ""
+    if music.get("external_ids") and len(music.get("external_ids")) > 0:
+        isrc = music.get("external_ids").get("isrc")
+    elif music.get("isrc"):
+        isrc = music.get("isrc")
+    # was a list with a singular entry for a while back in 2021
+    if isinstance(isrc, list):
+        isrc = isrc[0]
+    # some records contain the "ISRC" prefix that is described as legacy
+    # in the ISRC handbook from IFPI.
+    if isrc and isrc[:4] == "ISRC":
```

Comment:
`ISRC.parse()` seems to cover this case. I am not sure how it behaves when spaces are involved though.

Reply:
only when the ISRC is semi well-formed with a space between the prefix and ISRC (we don't have any of those in our dataset tho).

```python
>>> from iso3901 import ISRC
>>> ISRC.parse("ISRC AA6Q72000047")
ISRC(owner='AA6Q7', year=20, designation=47, country=None)
>>> ISRC.parse("ISRCAA6Q72000047")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/hairmare/Documents/git.repos/rabe/suisa_sendemeldung/venv/lib64/python3.11/site-packages/iso3901/isrc.py", line 116, in parse
    owner, year, desig = cls._parse(_raw)
                         ^^^^^^^^^^^^^^^^
  File "/home/hairmare/Documents/git.repos/rabe/suisa_sendemeldung/venv/lib64/python3.11/site-packages/iso3901/isrc.py", line 95, in _parse
    raise ValueError(f'Unexpected character found for segment "{segment}"')
ValueError: Unexpected character found for segment "A6"
```